build.gradle

```diff
@@ -449,6 +449,14 @@ subprojects {
     }
   }
 
+  // remove test output from Test, IntegrationTest and UnitTest
```

Comment:
I would say "remove test output from all test types" so that this comment doesn't go stale if we add more test types.